Atomic/atomic.py

```diff
@@ -1113,6 +1114,29 @@ def set_debug(self):
         if self.args.debug:
             self.debug = True
 
+    def get_last_time_all_scanned(self):
+        """
+        Will read through the scan_summary.json file and get the most recent scan's time.
+        """
+        try:
```

Comment:
Looking closer, there is already a generic method.  It is get_all_vulnerable_info in atomic.py and you should use it.